tools/distpartitioning/parmetis_wrapper.py

```diff
@@ -0,0 +1,331 @@
+import os
```

Comment:
Please polish this file after address the comment in the previous one, thx!

Reply:
+1, pls remove unused code...

Reply:
Done.